multitier/settings.py

```diff
@@ -39,7 +39,8 @@
     'ROUTER_APPS': ('auth', 'sessions', 'contenttypes'),
     'ROUTER_TABLES': [],
     'DEBUG_SQLITE3_PATHS': [],
-    'THEMES_DIRS': [os.path.join(settings.BASE_DIR, 'themes')]
+    'THEMES_DIRS': [os.path.join(settings.BASE_DIR, 'themes')],
+    'STATICS_DIRS': settings.STATICFILES_DIRS
```

Comment:
Everywhere in Django it uses the prefix ``STATICFILES_*``. Why ``STATICS_`` instead?